standalone/pherry/src/lib.rs

```diff
@@ -47,9 +47,9 @@ pub use phaxt::connect as subxt_connect;
 #[clap(
     about = "Sync messages between pruntime and the blockchain.",
     version,
-    author
+    author,
+    next_display_order = None
```

Comment:
Does `next_display_order = None` mean keeping the defining order?

Reply:
Looks like it defaults to `DeriveDisplayOrder` in 4.0.
https://github.com/clap-rs/clap/pull/3975
Let's just remove the `next_display_order = None`.